packages/salesforcedx-utils-vscode/package.json

```diff
@@ -17,16 +17,20 @@
     "@types/mocha": "^2.2.38",
```

Comment:
You added to salesforcedx-utils-vscode. Do you need to also remove from the other package.json?

Reply:
I only moved a couple of classes and their respective tests. There are still a ton of tests (test, test:unit and test:integration) in salesforcedx-apex-debugger that require it.